vhdbuilder/packer/install-dependencies.sh

```diff
@@ -410,7 +410,7 @@ start_watch
 
 # Kubelet credential provider plugins
 CREDENTIAL_PROVIDER_VERSIONS="
-1.29.3
+1.29.2
```

Comment:
windows binary of 1.29.3 is different from global and mooncake for cdn cache issue.
We use 1.29.2 instead, which has the same code as 1.29.3